From b2d530e10864ed6dfff6705ca4704ff4b59b5ec6 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Mon, 29 Aug 2022 23:27:53 +0200
Subject: [PATCH 440/440] arm64: dts: rk3399-pinebook-pro: Fix codec frequency
 after boot

Better fix. (for squashing into the previous patch with the same subject)

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
index 39a1c144e070..65e05d31812d 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
@@ -688,8 +688,8 @@ es8316: es8316@11 {
 		reg = <0x11>;
 		clocks = <&cru SCLK_I2S_8CH_OUT>;
 		clock-names = "mclk";
-		assigned-clocks = <&cru SCLK_I2S0_8CH>;
-		assigned-clock-rates = <11289599>;
+		assigned-clocks = <&cru SCLK_I2S_8CH_OUT>, <&cru SCLK_I2S1_8CH>;
+		assigned-clock-rates = <11289599>, <11289599>;
 		#sound-dai-cells = <0>;
 	};
 };
-- 
2.35.3

